iNaturalist.xcodeproj/project.pbxproj

```diff
@@ -1740,7 +1743,7 @@
 				);
 				PRODUCT_NAME = iNaturalist;
 				PROVISIONING_PROFILE = "";
-				"PROVISIONING_PROFILE[sdk=iphoneos*]" = "";
+				"PROVISIONING_PROFILE[sdk=iphoneos*]" = "0cd98630-25f1-43a1-bc92-c9b7214fabc2";
```

Comment:
Nobody other than you and your dev team will have this Provisioning Profile, so please don't include it in your pull request.